### DISCUSSION TITLE
Ppui canceled by you

### DISCUSSION THREAD

**Discussion by onkarsinghsahil** [ROM-builders/temporary#8983] [Q&A]
Sir why I can't built ppui
Problem in my repo ????

**Comment by Apon77** [comment]
> Sir why I can't built ppui Problem in my repo ????

are you sure? i didnt cancel any ppui build as far as i know

**Reply by onkarsinghsahil** [reply]
Sir Can I pm u in telegram

**Reply by Apon77** [reply]
> Sir Can I pm u in telegram

use group [@rombuilders](https://t.me/ROM_builders)
